orchestrator/api/system.py

```
@@ -335,10 +335,143 @@ async def get_system_health(db: Session = Depends(get_db)):
+            # Verify RAG service and its database connectivity
+            from modules.rag import get_rag_service
+            rag_service = get_rag_service()
+            
+            # Verify we can access RAG configurations
+            from core.models import RAGConfiguration
+            rag_config_count = db.query(RAGConfiguration).count()
+            
+            # Verify we can access document chunks (if table exists)
+            try:
+                chunk_count = db.execute("SELECT COUNT(*) FROM document_chunks").scalar()
+            except:
```

Ruff (0.14.7)

Do not use bare `except`

(E722)